src/WebJobs.Extensions.ApiHub/Table/TableClientValueProvider.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Reflection;
+using Microsoft.Azure.WebJobs.Host.Bindings;
+
+namespace Microsoft.Azure.WebJobs.Extensions.ApiHub.Table
+{
+    internal class TableClientValueProvider : IValueProvider
```

Comment:
nit: might consider making this a private class of the binding. we do that for the other bindings to reduce the number of files.